laravel/app/Models/Assignment.php

```diff
@@ -19,4 +19,29 @@ class Assignment extends Model
      * @var bool
      */
     public $timestamps = false;
+
+    public function residents()
```

Comment:
should be singular `resident`, since an assignment only belongs to one Resident.

Reply:
Singular relationships should be fixed next push!

Reply:
Great! Do you mind going through and resolving the conversations that have been addressed so I know they've been handled?